tests/test_h5store.py

```diff
@@ -690,23 +683,24 @@ def test_single_writer_multiple_reader_same_process(self):
                         assert reader2["test"]
 
     @pytest.mark.skipif(
-        WINDOWS, reason="This test fails for an unknown reason on Windows."
+        python_implementation() != "CPython", reason="SWMR mode not available."
     )
     def test_single_writer_multiple_reader_same_instance(self):
         from multiprocessing import Process
 
+        def _read_from_h5store(filename, **kwargs):
+            with H5Store(self._fn_store, mode="r", swmr=True) as h5s:
+                list(h5s)
+
         def read():
-            p = Process(
-                target=_read_from_h5store,
-                args=(self._fn_store,),
-                kwargs=(dict(mode="r")),
-            )
+            p = Process()
```

Comment:
I'm confused. Does this process _do_ anything? It seems like it just gets created and joined.

Reply:
Well that's strange. I'm not sure how it got pushed in this weird intermediate state, my fault for context switching between multiple branches. I've pushed the actual fix now.